### PR TITLE
change how names are created

### PR DESCRIPTION
**Pull request by mattkosack**
Needs to be reviewed.

### CODE REVIEW

**Comment by bjcoleman** [comment]
I can verify that this works on "old" Docker installs.

File: mirrulations-dashboard/src/mirrdash/dashboard_server.py [hunk 1]
```diff
@@ -28,13 +28,18 @@ def get_jobs_stats(database):
     }
 
 
+def get_container_name(container):
+    name = container.name.replace('-', '_')
```
Comment (bjcoleman) [inline mirrulations-dashboard/src/mirrdash/dashboard_server.py:36]: Add a comment on this line explaining the `-` vs. `_` issue with Docker.

**Comment by bjcoleman** [comment]
Once we add the comment I suggested, this is ready to go.  Someone else can do a quick review and then merge.

File: mirrulations-dashboard/src/mirrdash/dashboard_server.py (hunk 1 above)
Comment (mmorykan) [reply mirrulations-dashboard/src/mirrdash/dashboard_server.py:36]: Comment added

**Pull request merged**